crates/shared/src/price_estimation.rs

```diff
@@ -44,13 +44,15 @@ pub struct NativePriceEstimators(Vec<Vec<NativePriceEstimator>>);
 pub enum NativePriceEstimator {
     Driver(ExternalSolver),
     OneInchSpotPriceApi,
+    CoinGeckoProPriceApi,
```

Comment:
nit: Let's use short names where they don't add any more clarity

```suggestion
    CoinGecko,
```